.travis.yml

```diff
@@ -4,14 +4,14 @@ node_js:
 env:
   global:
   - CXX=g++-4.8
-  - APPLITOOLS_BATCH_ID=`echo ${TRAVIS_PULL_REQUEST_SHA:=$TRAVIS_JOB_ID}`
+#  - APPLITOOLS_BATCH_ID=`echo ${TRAVIS_PULL_REQUEST_SHA:=$TRAVIS_JOB_ID}`
   matrix:
   # - TEST_SUITE='build:ci' # Moved to Circle CI
   - TEST_SUITE='gemini set1'
   - TEST_SUITE='gemini set2'
   - TEST_SUITE='gemini set3'
   - TEST_SUITE='gemini set4'
-  - TEST_SUITE='applitools:ci'
+#  - TEST_SUITE='applitools:ci'
```

Comment:
nit: why commented instead of removed?

Reply:
An escape hatch in case we needed to quickly go back to previous configuration.